### PR TITLE
Add support for stylesheets and ERB views to `rails stats`.

### PR DESCRIPTION
**Pull request by joelhawksley**
### Summary

In looking for ways to report statistics on the size of GitHub's monolith, I realized that `rails stats` does not report on the `app/views` and `app/assets/stylesheets` directories. 

I think folks would find it useful to see stats on these folders. This PR adds them as `Views` and `Stylesheets`:

```console
+----------------------+--------+--------+---------+---------+-----+-------+
| Name                 |  Lines |    LOC | Classes | Methods | M/C | LOC/M |
+----------------------+--------+--------+---------+---------+-----+-------+
| Controllers          |      6 |      6 |       2 |       1 |   0 |     4 |
| Helpers              |      2 |      2 |       0 |       0 |   0 |     0 |
| Jobs                 |      2 |      2 |       1 |       0 |   0 |     0 |
| Models               |     35 |     29 |       3 |       6 |   2 |     2 |
| Mailers              |      4 |      4 |       1 |       0 |   0 |     0 |
| Views                |    130 |    127 |       0 |       0 |   0 |     0 |
| JavaScripts          |     16 |      0 |       0 |       0 |   0 |     0 |
| Stylesheets          |     21 |     19 |       0 |       0 |   0 |     0 |
+----------------------+--------+--------+---------+---------+-----+-------+
| Total                |    216 |    189 |       7 |       7 |   1 |    25 |
+----------------------+--------+--------+---------+---------+-----+-------+
  Code LOC: 189     Test LOC: 0     Code to Test Ratio: 1:0.0
```

### CODE REVIEW

File: railties/CHANGELOG.md [hunk 1]
```diff
@@ -1,8 +1,11 @@
+*   Add support for stylesheets and ERB views to `rails stats`.
+
+    *Joel Hawksley*
+
 *   Allow appended root routes to take precedence over internal welcome controller.
 
     *Gannon McGibbon*
 
-
```
Comment (rafaelfranca) [inline railties/CHANGELOG.md]: This line was correct, can you put it back please?

**Review by rafaelfranca — COMMENTED** [review]
Looks good. Just a small comment about the changelog. Can you squash your commits in only one?

File: railties/CHANGELOG.md (hunk 1 above)
Comment (joelhawksley) [reply railties/CHANGELOG.md]: Will do once I figure out this rebase I just messed up ❤️

**Comment by joelhawksley** [comment]
@rafaelfranca alright, I squashed to a single commit.

I pointed this at 6-1-stable, but happy to point to `master` if you'd prefer to leave it out of `6.1` ❤️

**Comment by kaspth** [comment]
All features go to the master branch and then we backport if suitable.

**Comment by joelhawksley** [comment]
@rafaelfranca @kaspth sorry for all the hassle for such a small PR! This should be all cleaned up now, pointing to `master`.

**Comment by joelhawksley** [comment]
Fixing the failing test now.

**Comment by joelhawksley** [comment]
@rafaelfranca fixed!

**Comment by joelhawksley** [comment]
@rafaelfranca this should be good to go ❤️

**Comment by kaspth** [comment]
Sweet, thanks @joelhawksley!

**Pull request merged**